addons/social_share/views/snshare_frontend.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
```

Comment:
:socks: we generally use a "templates" for file name, like ``trombinoscope_templates_frontend.xml``